Content.Shared/Contraband/ShowContrabandDetailsComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+namespace Content.Shared.Contraband;
+
+/// <summary>
+///     This component allows you to see Contraband details on examine items
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ShowContrabandDetailsComponent : Component
+{
+
+}
```

Comment:
Empty body is redundant

```suggestion
public sealed partial class ShowContrabandDetailsComponent : Component;
```